examples/raspberry-pi.rs

```diff
@@ -1,4 +1,4 @@
-use navigator_rs::{pwm_Channel, Navigator, SensorData};
+use navigator_rs::{Navigator, PwmChannel};
```

Comment:
Pretty sure the commit message should be `examples: Update PwmChannel and remove unneeded module`